src/model/document/document.js

```diff
@@ -218,6 +230,10 @@ class Document {
     return this.rwMeta('text', newText);
   }
 
+  reference(){
```

Comment:
I intend to call this 'citation' which returns UI-friendly strings like `{ title, authors, reference }`